src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -11,6 +11,7 @@ export default (accessToken, userID, opts) => {
   )
 
   return {
+    artworkAuthLoader: gravityLoader(id => `artwork/${id}`),
```

Comment:
Any reason why we can't use the same name as the loader without authentication? Looks like we have the pattern for sale artworks loader ([authenticated](https://github.com/artsy/metaphysics/blob/0a8c2e258da328357346774ebbf23379d8577d08/src/lib/loaders/loaders_with_authentication/gravity.js#L103), [unauthenticated](https://github.com/artsy/metaphysics/blob/0a8c2e258da328357346774ebbf23379d8577d08/src/lib/loaders/loaders_without_authentication/gravity.js#L86)) already. _I think_ it will simply [overwrite](https://github.com/artsy/metaphysics/blob/0a8c2e258da328357346774ebbf23379d8577d08/src/lib/loaders/index.js#L16-L19) and use the authenticated one when there is `access_token` present. But I could miss something here.

Reply:
👍 yup that's exactly what will happen!